hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -448,6 +449,19 @@ public static boolean fileExists(FileSystem fs, Path path) {
     }
   }
 
+  /**
+   * Returns whether the given instant is a data writing commit.
+   *
+   * @param tableType The table type
+   * @param instant   The instant
+   * @param timeline  The timeline
+   */
+  public static boolean isWriteCommit(HoodieTableType tableType, HoodieInstant instant, HoodieTimeline timeline) {
+    return tableType == HoodieTableType.MERGE_ON_READ
```

Comment:
Is the replacecommit generated from `INSERT OVERWRITE` write commit?

Reply:
We already checked the clustering plan.

Reply:
The replacecommit generated by clustering and insert overwrite is different.